tools/c7n_gcp/c7n_gcp/resources/resourcemanager.py

```diff
@@ -24,12 +27,14 @@ class resource_type(TypeInfo):
         component = 'organizations'
         scope = 'global'
         enum_spec = ('search', 'organizations[]', {'body': {}})
+        get_requires_event = True
         id = "name"
 
         @staticmethod
-        def get(client, resource_info):
+        def get(client, event):
```

Comment:
organization probably shouldnt have a get method, there's no associated lifecycle event to it. ie. the test case here is using a project lifecycle event.

Reply:
Yep, I see, makes sense as I cannot make up a use case other than receiving the "Project added to your organization" notification. Removed the method.